cmd/tuf/app/add-delegation.go

```diff
@@ -77,21 +86,38 @@ but will default to the name if unspecified.
 			if len(keys) == 0 {
 				return flag.ErrHelp
 			}
-			return DelegationCmd(ctx, *repository, *name, *path, *terminating, keys, *targets)
+			if len(keys) < *threshold {
+				return flag.ErrHelp
+			}
+			opts := DelegationOptions{
+				Directory:   *repository,
+				Name:        *name,
+				Path:        filepath.Join(*name, "*"),
+				Terminating: true,
+				KeyRefs:     []string(keys),
+				Threshold:   *threshold,
+				Targets:     *targets,
+			}
+			return DelegationCmd(ctx, &opts)
 		},
 	}
 }
 
-func DelegationCmd(ctx context.Context, directory, name, path string, terminating bool, keyRefs keysFlag, targets string) error {
-	store := tuf.FileSystemStore(directory, nil)
+func DelegationCmd(ctx context.Context, opts *DelegationOptions) error {
```

Comment:
I will file an issue here. Per the doc we discussed, we'll need to require updating this delegation stub at least once a year (assuming no other rotation) - I believe the CLI will error out trying to add a new delegation when it already exists, or at least I ran into this problem before (see issue https://github.com/theupdateframework/go-tuf/issues/330). 

Obviously it won't block initial creation of the delegation, just important to follow-up on it.

Reply:
https://github.com/sigstore/root-signing/issues/612

Reply:
I'll probably take a stab at #612 as part of finishing this.

Reply:
Resolving this because we're tabling for a future PR improvement!